core/src/main/java/org/jdbi/v3/core/transaction/LocalTransactionHandler.java

```diff
@@ -70,6 +70,7 @@ public void rollback(Handle handle) {
 
     @Override
     public void savepoint(Handle handle, String name) {
+        @SuppressWarnings("PMD.CloseResource")
```

Comment:
Would be nice if we could annotate `Handle.getConnection()` so that PMD knows the connection wasn't created for this method.

Reply:
I don't think such a thing exists. There's `@Tainted` et al but I don't think there's anything for "this is a Resource but it doesn't need to be closed"...